Userland/Services/WindowServer/WindowFrame.cpp

```diff
@@ -535,6 +535,40 @@ void WindowFrame::layout_buttons()
         m_buttons[i].set_relative_rect(button_rects[i]);
 }
 
+bool WindowFrame::hit_test(const Gfx::IntPoint& point) const
+{
+    if (m_window.is_frameless())
+        return false;
+    auto frame_rect = rect();
+    if (!frame_rect.contains(point))
+        return false;
+    auto window_rect = m_window.rect();
+    if (window_rect.contains(point))
+        return false;
+
+    u8 alpha_threshold = Gfx::WindowTheme::current().frame_alpha_hit_threshold(window_state_for_theme()) * 255;
+    if (alpha_threshold == 0)
+        return true;
```

Comment:
What is the difference between threshold 0 and 1?

Reply:
Oh wait, I guess 0 just means that no alpha hit testing should be done at all.

Reply:
That's right, a threshold of 0 would mean always hit, ignoring the alpha channel